src/pseudoflow/core/libhpf.c

```diff
@@ -1440,13 +1511,13 @@ static void destroyProblem(CutProblem *problem, int destroySourceSetIndicator)
 destroyProblem - Destruct function for CutProblem struct
 *************************************************************************/
 {
-	free(problem->sourceSet);
+	//free(problem->sourceSet);
```

Comment:
Have you confirmed that no memory leaks were introduced with e.g. valgrind?
Please delete any code that is no longer needed.